packages/razzle/scripts/build.js

```diff
@@ -74,8 +74,12 @@ function build(previousFileSizes) {
   let razzle = {};
   try {
     razzle = require(paths.appRazzleConfig);
+    console.log(chalk.yellow('Found custom razzle.config.js'));
     /* eslint-disable no-empty */
-  } catch (e) {}
+  } catch (e) {
+    const errMessage = chalk.yellow(e.message);
+    console.log(`No custom razzle config processed (${errMessage})`);
```

Comment:
The problem with this though is that this will _always_ show if there is no `razzle.config.js` (ie. the require() fails).